aws/data_aws_unity_catalog_assume_role_policy.go

```diff
@@ -14,13 +15,19 @@ func DataAwsUnityCatalogAssumeRolePolicy() common.Resource {
 		UnityCatalogIamArn string `json:"unity_catalog_iam_arn,omitempty" tf:"computed"`
 		ExternalId         string `json:"external_id"`
 		AwsAccountId       string `json:"aws_account_id"`
+		AwsPartition       string `json:"aws_partition,omitempty" tf:"default:aws"`
 		JSON               string `json:"json" tf:"computed"`
 		Id                 string `json:"id" tf:"computed"`
 	}
 	return common.NoClientData(func(ctx context.Context, data *AwsUcAssumeRolePolicy) error {
 		if data.UnityCatalogIamArn == "" {
 			data.UnityCatalogIamArn = "arn:aws:iam::414351767826:role/unity-catalog-prod-UCMasterRole-14S5ZJVKOTYTL"
 		}
+
+		if !slices.Contains(common.AwsPartitions, data.AwsPartition) {
+			return fmt.Errorf("aws_partition must be either 'aws', 'aws-cn', or 'aws-us-gov'")
```

Comment:
nit: we can move error message into constants...

Reply:
Done 🙌